apps/ex_nvr_web/lib/ex_nvr_web/controllers/api/user_controller.ex

```diff
@@ -0,0 +1,63 @@
+defmodule ExNVRWeb.API.UserController do
+  @moduledoc false
+
+  use ExNVRWeb, :controller
+
+  action_fallback ExNVRWeb.API.FallbackController
+
+  plug ExNVRWeb.Plug.User, [field_name: "id"] when action in [:update, :show, :delete]
```

Comment:
Unlike the `device` where a lot of endpoints relies on it. The `user` id in the endpoints is only used in this controller. I suggest you make this plug local to this controller.

Reply:
What I mean by local is to make a local function as plug. No need to copy the whole module.